src/utils/openvpn.ts

```diff
@@ -387,22 +385,22 @@ export class VpnManager extends EventEmitter {
 				.then(() => {
 					this.emit('manager:connect');
 				}),
-		).return(true);
+		).return();
 	}
 
-	public exec(command: string): Bluebird<true> {
-		return Bluebird.try(() => this.connector.exec(command)).return(true);
+	public async exec(command: string): Promise<void> {
+		return Bluebird.try(() => this.connector.exec(command)).return();
 	}
 
-	public enableLogging() {
+	public async enableLogging() {
 		return this.exec('log on all');
 	}
 
-	public enableBytecountReporting(interval: number) {
+	public async enableBytecountReporting(interval: number) {
 		return this.exec(`bytecount ${interval}`);
 	}
 
-	public releaseHold() {
+	public async releaseHold() {
```

Comment:
There's no need to mark these as `async` if all they do is call another promise-returning function - as I understand it functionally it will just cause the promise to be wrapped, ie it's equivalent to doing `return Promise.resolve(this.exec('hold release'))`